contrib/src/main/java/com/datatorrent/contrib/twitter/TwitterSampleInput.java

```diff
@@ -204,6 +205,13 @@ public void onException(Exception ex)
     }
   }
 
+  /**
+   * Allow derived classes to customize the configuration
+   */
+  protected void setupConfigurationBuilder(ConfigurationBuilder cb)
+  {
```

Comment:
Would be better to move the configuration code here from setup(); derived classes would then have the option to invoke or ignore the superclass method.